SUPPORTED.md

```diff
@@ -11,7 +11,6 @@
 |  Protocol  |  MessageLine Address  |
 |------------|--------------------------------------------|
 | ORMP       | 0x0000000000D2de3e2444926c4577b0A59F1DD8BC |
-| ORMPExt    | 0x000000000EFcBAdA3793cC59c62D79b9f56Ae48F |
```

Comment:
I have noticed this change today and the page at https://docs.darwinia.network/supported-networks-c44e644252f7484495b5a4d65ba772db should also be updated. Please raise an issue https://www.notion.so/darwinia/69272707ca3e44a3a3ff1e0d4053800a?v=d2fb87001bf846d3b41949b547c4b8c4.

Suggest that take the doc into consideration in development, it's really important to maintain a high quality document as the darwinia msgport is now one of our major products. I am working on a msgport document improvement plan recently, and need the support from the msgpor team.